lib/fixup.rb

```diff
@@ -215,8 +233,8 @@ def self.postremove; end
 FileUtils.rm "#{CREW_PACKAGES_PATH}/pagerenv" if File.file?("#{CREW_PACKAGES_PATH}/pagerenv")
 
 # Handle broken system glibc affecting gcc_lib on newer x86_64 ChromeOS milestones.
-if (ARCH == 'x86_64') && (LIBC_VERSION.to_f >= 2.36)
-  abort("patchelf is needed. Please run: 'crew install patchelf && crew update'") unless File.file?(File.join(CREW_PREFIX, 'bin/patchelf'))
+if (ARCH == 'x86_64') && (LIBC_VERSION >= '2.36')
```

Comment:
Better to compare numbers than strings?

Reply:
Apparently it works for both. Is it faster not to have to do a conversion to float?

Reply:
Here is the potential problem with string comparisons of version numbers.
```
irb(main):005> str1 = '2.4'
=> "2.4"
irb(main):006> str2 = '2.36'
=> "2.36"
irb(main):007> str1 > str2
=> true
```
But is it really true for versions?

Reply:
That's fair. I'll change it.